.github/workflows/python-package.yml

```diff
@@ -88,6 +88,63 @@ jobs:
         cd ${{ env.OTIO_CONSUMER_TEST_BUILD_DIR }}
         cmake ${{ github.workspace }}/tests/consumer -DCMAKE_PREFIX_PATH=${{ env.OTIO_INSTALL_DIR }}
 
+  cpp_build_external_deps:
```

Comment:
A comment describing this job and what distinguishes this from the simple `cpp_build` job would be nice. 
 With three jobs, it feels like we would want comments on the other two as well, but that is outside the scope of thisPR.